### PR TITLE
Footer for egi whitelabel

### PR DESCRIPTION
**Pull request by jarekzet**
new footer layout for EGI whitelabel
https://github.com/cyfronet-fid/marketplace-egi/issues/3
https://marketplace.egi.eu/

### CODE REVIEW

**Comment by Marcelinna** [comment]
we should delete element below footer ?
image

**Comment by Marcelinna** [comment]
ticket
image

PR 
image

In PR is additionally added youtube, twiter logo - which redirect to EOSC media
In PR is added a link to the EGI logo which redirect to the EOSC page (https://eosc-portal.eu/)

should these elements be introduced?

**Comment by lukasz-korzeniowski** [comment]
Twiter should be linked to: https://twitter.com/egi_einfra 
Please replace Youtube  with LinkedIn icon and link to: https://www.linkedin.com/company/egi-foundation

**Comment by jarekzet** [comment]
Added mods on social media links:
Twiter: https://twitter.com/egi_einfra
LinkedIn: https://www.linkedin.com/company/egi-foundation

Removed link on logo in footer.

**Comment by Marcelinna** [comment]
So we need link to: 
Contact us
Terms and conditions
SLA
Sitemap 
 My orders
My credit slips
My addresses
My personal info 
Privacy policy

File: app/views/layouts/_navbar.html.haml [hunk 1]
```diff
@@ -7,17 +7,17 @@
...
           %li
             %a.simple-item{ href: "/favourites" }= _("Favourite Resources")
           %li
             %a.simple-item{ href: "/profile" }= _("Profile")
           %li
-            = link_to _("Logout"), destroy_user_session_path, method: :delete, "data-e2e" => "logout",
-            class: "btn btn-primary font-weight-bold mt-5 logout-link"
+            = link_to _("Logout"), destroy_user_session_path, method: :delete,
+            class: "btn btn-primary font-weight-bold mt-5"
         - else
```
Comment (Marcelinna) [inline app/views/layouts/_navbar.html.haml:18]: why you delete logout-link?

File: app/views/layouts/_sections.html.haml [hunk 2]
```diff
@@ -21,7 +21,7 @@
           %i.fas.fa-user.mr-1
           = _("#{current_user.first_name}" "#{current_user.last_name}")
         %li
-          %strong= link_to _("Logout"), destroy_user_session_path, method: :delete
+          %strong= link_to _("Logout"), destroy_user_session_path, method: :delete, "data-e2e" => "logout"
       - else
         %li
-          %strong= link_to _("Login"), user_checkin_omniauth_authorize_path
```
Comment (Marcelinna) [inline app/views/layouts/_sections.html.haml:27]: Please delete "data-e2e" => "my-eosc-button"

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**